analysis_driver/pipelines/common.py

```diff
@@ -69,6 +70,24 @@ def bcbio_prepare_sample(job_dir, sample_id, fastq_files):
     return sample_fastqs
 
 
+def get_genome_version(dataset_name, species):
+    genome_version = clarity.get_sample(dataset_name).udf.get('Genome Version')
+    if genome_version is None:
+        genome_version = cfg.query('species', species, 'default')
+    reference = cfg.query('genomes', genome_version, 'fasta')
+    if not reference:
+        raise PipelineError('Could not find reference for species %s in sample %s ' % (species, dataset_name))
+    return genome_version, reference
+
+
+def get_dbsnp(genome_version):
+    return cfg.query('genomes', genome_version, 'dbsnp')
+
+
+def get_known_indels(genome_version):
+    return cfg.query('genomes', genome_version, 'known_indels')
```

Comment:
Do we really need these functions? They don't really do much themselves and are only called once each in variant_calling